Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -403,6 +403,9 @@ public class LobbyRTCState
 
         // Has this person enabled press to talk
         public bool PressToTalkEnabled = false;
+
+        // We have locally blocked this person from receiving and sending audio
+        public bool isBlocked = false;
```

Comment:
The _only_ thing that I believe needs to be changed before this can be approved is that (for the sake of consistency with our team's coding standards and for the other field members within `LobbyRTCState`) this field member should be capitalized since it is public (it should be changed to `IsBlocked`).